new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/MySQLDBUtils.java

```diff
@@ -0,0 +1,130 @@
+import io.kubernetes.client.openapi.models.V1Pod;
+import io.kubernetes.client.openapi.models.V1PodSpec;
+import io.kubernetes.client.openapi.models.V1Secret;
+import io.kubernetes.client.openapi.models.V1SecretKeySelector;
+import io.kubernetes.client.openapi.models.V1Service;
+import io.kubernetes.client.openapi.models.V1ServicePort;
+import io.kubernetes.client.openapi.models.V1ServiceSpec;
+import oracle.weblogic.kubernetes.actions.TestActions;
+import oracle.weblogic.kubernetes.actions.impl.Namespace;
+import oracle.weblogic.kubernetes.actions.impl.primitive.Kubernetes;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * Utility class to create MySQL database.
+ */
+public class MySQLDBUtils {
+
+  /**
+   * Create and start a MySQL database pod.
+   *
+   * @param name      name of the db pod
+   * @param user      username for the database
+   * @param password  password for the database
+   * @param nodePort  node port of db service
+   * @param namespace name of the namespace in which to create MySQL database
+   */
+  public static void createMySQLDB(String name, String user, String password, int nodePort, String namespace) {
+
+    String uniqueName = Namespace.uniqueName();
+    String secretName = name.concat("-secret-").concat(uniqueName);
+    String serviceName = name.concat("-external-").concat(uniqueName);
+
+    createSecret(secretName, user, password, namespace);
+    createService(serviceName, name, namespace, nodePort);
+    startMySQLDB(name, secretName, namespace);
+
+  }
+
+  //create the database pod
+  private static void startMySQLDB(String name, String secretName, String namespace) {
+    Map<String, String> labels = new HashMap<>();
+    labels.put("app", name);
+    V1Pod mysqlPod = new V1Pod()
+        .metadata(new V1ObjectMeta()
+            .name(name)
+            .namespace(namespace)
+            .labels(labels))
+        .spec(new V1PodSpec()
+            .terminationGracePeriodSeconds(5L)
+            .containers(Arrays.asList(new V1Container()
+                .image("mysql:5.6")
+                .name("mysql")
+                .addEnvItem(new V1EnvVar()
+                    .name("MYSQL_ROOT_PASSWORD")
+                    .valueFrom(new V1EnvVarSource()
+                        .secretKeyRef(new V1SecretKeySelector()
+                            .name(secretName)
+                            .key("root-password"))))
+                .ports(Arrays.asList(new V1ContainerPort()
+                    .name("mysql")
+                    .containerPort(3306))))));
+    V1Pod pod = assertDoesNotThrow(() -> Kubernetes.createPod(namespace, mysqlPod));
+    CommonTestUtils.checkPodReady(pod.getMetadata().getName(), null, namespace);
+  }
+
+  //create services for MySQL database
+  private static void createService(String serviceName, String selectorName, String namespace, int port) {
```

Comment:
Change it to createDBService? When called from a test, it will be clear that a db service is being created.

Reply:
fixed.